src/frequenz/sdk/timeseries/ev_charger_pool/_ev_charger_pool.py

```diff
@@ -158,6 +165,40 @@ async def component_data(self, component_id: int) -> Receiver[EVChargerData]:
 
         return output_chan.new_receiver()
 
+    async def set_bounds(self, component_id: int, max_amps: float) -> None:
+        """Send given max current bound for the given EV Charger to the microgrid API.
+
+        Bounds are used to limit the max current drawn by an EV, although the exact
+        value will be determined by the EV.
+
+        Args:
+            component_id: ID of EV Charger to set the current bounds to.
+            max_amps: Current bound value to set for the EV Charger.
```

Comment:
maybe `Max current bound value to set for the EV charger` ?

Reply:
The word `bound` already implies `max`.  Because we're only dealing with charging,  there's only positive values.  So I'd think `max amps == current bound`.

Reply:
I've added a new commit replacing the docstring with: `maximum current in amps, that an EV can draw from this EV Charger.`  does that look better?

Reply:
yes, it  does. Thank you!